drape_frontend/stylist.cpp

```diff
@@ -227,16 +227,31 @@ class Aggregator
 IsHatchingTerritoryChecker::IsHatchingTerritoryChecker()
 {
   Classificator const & c = classif();
-  base::StringIL arr[] = {
+
+  base::StringIL arr3[] = {
+    {"boundary", "protected_area", "1"},
```

Comment:
Could you please add a comment that it'll match all sub-types also.
ATM one has to read through the code to understand it..